nextjs-app/constants/groups.ts

```diff
@@ -0,0 +1,32 @@
+export const groupsInfo = [
+  {
+    zhGroupName: "商業開發",
+    enGroupName: "Business (BD)",
+    imageUrl: "/images/group-icon/bd.png",
+  },
+  {
+    zhGroupName: "資料應用",
+    enGroupName: "Data Application (Data)",
+    imageUrl: "/images/group-icon/data.png",
+  },
+  {
+    zhGroupName: "技術開發",
+    enGroupName: "Engineering",
+    imageUrl: "/images/group-icon/mkt.png",
```

Comment:
I can not download the icon for engineering group for now, need to wait for the response from designer

Reply:
updated in d73e84750422536a326dcbfe594dc8af211fa0e0